kotlin/internal/jvm/jvm.bzl

```diff
@@ -161,6 +161,11 @@ _common_attr = utils.add_dicts(
             default = [],
             allow_files = False,
         ),
+        "friend": attr.label(
```

Comment:
The internal rules uses "friends". Please continue to use the plural form.

Reply:
Sigh. Ok. So if we do that, then we really should not have the one-friend-only restriction (e.g., friends-graph rather than friends-tree) otherwise this is terribly named. :/  But either way, I guess conformity iwth the internal is worth it.